sections/footer.liquid

```diff
@@ -75,19 +75,43 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
```

Comment:
I wonder if we should/could re organize the general settings to match the output order: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/11-14-srabo-f4nlg.png)
</details>

Seems like brand info is only used in the footer so far 🤔 So it should be ok

Reply:
cc @allylane